daprdocs/data/components/state_stores/azure.yaml

```diff
@@ -1,6 +1,6 @@
 - component: Azure Blob Storage
   link: setup-azure-blobstorage
-  state: Stable
```

Comment:
LGTM. Can we also make the changes in 1.8 branch for the components that are stable candidates already?